utility/mm2_crawler

```diff
@@ -748,15 +748,15 @@ def sync_hcds(session, host, host_category_dirs, repodata):
 
 def method_pref(urls, prev=""):
     """ return which of the hosts connection method should be used
-    rsync > http > ftp """
+    rsync > http(s) > ftp """
     pref = None
     for u in urls:
         if prev.startswith('rsync:'):
             break
         if u.startswith('rsync:'):
             return u
     for u in urls:
-        if u.startswith('http:'):
```

Comment:
I would prefer a "if u.startswith(('http:', 'https:'))", so that we have the colon included at least, so we know it's a protocol indication.